README.md

```diff
@@ -0,0 +1,16 @@
+# Jukebox
+
```

Comment:
Short description of the software maybe?

Reply:
Done